app/src/autocomplete.js

```diff
@@ -128,10 +138,13 @@ class Autocomplete {
     return Math.floor(inputWidth / 35);
   }
 
-  _source(params, locale) {
+  _source(params, locale, clickAnalyticsEnabled) {
     return (query, callback) => {
-      this.index.search({...params, query, optionalWords: getOptionalWords(query, locale)})
-        .then((content) => { callback(this._reorderedHits(content.hits)); });
+      this.index.search({...params, clickAnalytics: clickAnalyticsEnabled, query, optionalWords: getOptionalWords(query, locale)})
+        .then((content) => { 
+          const hits = this._addPositionToHits(content.hits, content.queryID);
+          callback(this._reorderedHits(hits));
```

Comment:
This is just a stylistic comment, but maybe you could call everything at once?

```js
callback(
  this._reorderedHits(
    this._addPositionToHits(content.hits, content.queryID)
  )
);
```

Or with multiple constants:

```js
  const hitsWithPosition = this._addPositionToHits(content.hits, content.queryID);
  const reorderedHits = this._reorderedHits(hitsWithPosition);
  callback(reorderedHits);
```